apps/library/include/graphic_screen_api.h

```diff
@@ -20,12 +20,12 @@
 #define VGA_PTR(PTR,OFF)   PTR    = (volatile int *) (VGA_MEM_BASE + OFF)
 
 /* Control registers addresses */
-#define CR_MEM_BASE 0x00002000
```

Comment:
This is where we should add the set cursor function.
Probeblt should add new type of printf that allows you to print to any location. 
rvc_printf_at_location(char* text, int line_cursor, int collum_cursor)

Reply:
I don't want to give responsibility to the user for the cursor initialization operation because it might be forgotten, so I put it in crt0.s for the avoidance of doubt. As for the function it already exist - "draw_symbol"

Reply:
Maybe by including the lib it will define the default value of the corsur?

Note: If I remember correctly, the corsur is a CR.. which has a reset value - which resets to 0

The draw_simbol will allow you to write string to any location on the display?

Reply:
Yes the cursor is CR. As for the draw symbol - YES.

Reply:
![image](https://user-images.githubusercontent.com/81047407/184347058-142f1e5c-e671-4aba-8c81-81cb8a181606.png)

This dos not look like you can print a string.. 
Would like a function that can printf to any location on screen.